config/routes.rb

```diff
@@ -87,6 +87,7 @@
       get '/account_reset/pending/confirm' => 'account_reset/pending#confirm'
       post '/account_reset/pending/cancel' => 'account_reset/pending#cancel'
 
+      get '/auth_method_confirmation' => 'mfa_confirmation#index'
```

Comment:
Looks like this method was added but here and so was the template, but I'm not seeing it in the diff. Can we add a stub for it like this?

https://github.com/18F/identity-idp/blob/5968911e6947ccd5ebaae3ebece9bf27945ef42c/app/controllers/users/backup_code_setup_controller.rb#L13

Reply:
so add that stub to the `mfa_confirmation_controller.rb` file? If so, that makes sense.

Reply:
This would also be where we could add a check to not render in production if we want to prevent it from being available there